### PR TITLE
CF-477: prefs svc catalog json

### PR DESCRIPTION
**Pull request by shintasmith**
This change adapts the xml2json.xslt to accommodate the new addition of Preference Service endpoint in the Feeds Catalog:
```
<?xml version="1.0" encoding="UTF-8"?>
<service xmlns="http://www.w3.org/2007/app" xmlns:atom="http://www.w3.org/2005/Atom">
   ...
   <workspace>
      <atom:title>ssl_usagesummary_events</atom:title>
      <collection href="https://atom-shintacentos-n01.dev.ord1.us.ci.rackspace.net/usagesummary/ssl/events">
         <atom:title>ssl_usagesummary_events</atom:title>
      </collection>
   </workspace>
   <workspace>
      <atom:title>Feeds Archiving Preferences Service endpoints</atom:title>
      <atom:link href="https://test.preferences.feeds.api.rackspacecloud.com/archive/"
                 rel="archive-preferences"/>
   </workspace>
   <!-- Generated from schema version 1.89.0 -->
</service>
```

### CODE REVIEW

File: src/test/scala/Xml2JsonSuite.scala [hunk 1]
```diff
@@ -268,6 +268,30 @@ class Xml2JsonSuite extends BaseUsageSuite {
         .get( "region") == "DFW")
     }
 
+    test("should transform Feeds Catalog with Prefs Svc workspace to JSON properly") {
+        val parser = (new JsonParserFactory()).createFastParser()
+
+        val transformResult = transform( new StreamSource( new File( "src/test/resources/feedscatalog.xml" ) ) )
+        val transformMap = parser.parseMap( new ByteArrayInputStream( transformResult.getBytes( StandardCharsets.UTF_8 ) ) );
+
+        val jsonMap = parser.parseMap( new FileInputStream( "src/test/resources/feedscatalog.json" ) )
```
Comment (usnavi) [inline src/test/scala/Xml2JsonSuite.scala]: I think you should be testing against transformMap, right?  

My just-checked-in-code had this same jsonMap line.  I had originally thought that I could diff the 2 maps and was unintentionally checking it against the wrong map.

Comment (shintasmith) [inline src/test/scala/Xml2JsonSuite.scala]: Hahaha. Funny!

**Comment by shintasmith** [comment]
@usnavi, I updated the test.

**Comment by usnavi** [comment]
lgtm - @ChandraAddala can merge.

**Comment by shintasmith** [comment]
@usnavi, @ChandraAddala is not here. Can I merge this?

**Comment by usnavi** [comment]
sure

**Pull request merged**